File: DynamoDB-SDK-Examples/python/WorkingWithTables/table_change_to_provisioned.py
from __future__ import print_function # Python 2/3 compatibility
import boto3, pprint

dynamodb = boto3.resource('dynamodb', region_name='us-west-2')

table = dynamodb.Table("Music") # Substitute your table name

table.update(
    ProvisionedThroughput={
        'ReadCapacityUnits': 10,
        'WriteCapacityUnits': 10
    }
)

File: DynamoDB-SDK-Examples/node.js/WorkingWithTables/table_change_to_provisioned.js
const AWS = require("aws-sdk");

const dynamodb = new AWS.DynamoDB({ region: "us-west-2" });

const updateTable = async () => {
  const response = await dynamodb
    .updateTable({
      BillingMode: "PROVISIONED",
      ProvisionedThroughput: {
        ReadCapacityUnits: 5,
        WriteCapacityUnits: 10,
      },
      TableName: "Music", // Substitute your table name for "Music"
    })
    .promise();

  await dynamodb.waitFor("tableExists", { TableName: "Music" }).promise();
  console.log("Table has been updated");
};

updateTable().catch((error) => console.error(JSON.stringify(error, null, 2)));
